### ISSUE TITLE
dummyfile.txt

### ISSUE THREAD

**Issue by dpanfili** [ryanaltvater/generator-liquid#1]
When I run yo liquid, and enable option (or disable for that matter) only a dummyfile.txt file is generated.

MacBook-Pro-2:tia-yo davepanfili$ yo liquid

```
 _-----_
|       |    .--------------------------.
|--(o)--|    |  Welcome to the splendid |
```

   `---------´   |     generator-liquid     |
    ( _´U`_ )    |        generator!        |
    /**_A**_\    '--------------------------'
     |  ~  |  
   **'._**.'__  
 ´   `|° ´ Y` 

? Would you like to enable this option? Yes
identical dummyfile.txt
MacBook-Pro-2:tia-yo davepanfili$

**Comment by utahisnotastate** [comment]
Hi! So I actually ran into this same problem, but I'm on a windows PC. I figured out how to resolve this, and I thought i'd post it and maybe you would know how to translate the instructions so it can work on your mac. 

So, I noticed after running npm install -g generator-liquid it said it installed version 0.0.0. 

I went to c:\users\username\AppData\Roaming\npm\node_modules\generator-liquid\generators and I saw that the only folder in that directory was app, and it had the dummyfile.txt. 

I then downloaded the generator zip file directly from here, and opened up the generator-liquid\generators folder and it had folders titled "app" "static" "wordpress". I coped these three folders, and pasted them back in c:\users\username\AppData\Roaming\npm\node_modules\generator-liquid\generators. 

There was a pop up saying "do you want to replace this file" and it was some js file, and I said yes. 

After the folders were copied over, the generator works, and i'm installing wordpress as we speak!

Hope this helps! Sorry it's for windows only, but maybe Macs have a similar folder that you could use instead?

**Comment by ryanaltvater** [comment]
Hey guys,

Thanks for checking out my generator. As of yesterday, I officially launched 1.0.0. I was new to creating my own Yeoman generator 2 weeks ago and was checking out some commands when I prematurely published 0.0.0 with nothing in it. Sorry for the inconvenience. You should now be able to update the generator locally and pull the latest without any errors or dummyfile.txt instances. Let me know if you run into any other issues. :)
